[2022-08-06 19:38:36,091] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-04-08T00:00:00+00:00 [queued]>
[2022-08-06 19:38:36,108] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-04-08T00:00:00+00:00 [queued]>
[2022-08-06 19:38:36,108] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 19:38:36,108] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2022-08-06 19:38:36,108] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 19:38:36,126] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): second_task> on 2022-04-08 00:00:00+00:00
[2022-08-06 19:38:36,134] {standard_task_runner.py:52} INFO - Started process 2484 to run task
[2022-08-06 19:38:36,137] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v5', 'second_task', 'scheduled__2022-04-08T00:00:00+00:00', '--job-id', '764', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmppw20ksf9', '--error-file', '/tmp/tmpypm58z04']
[2022-08-06 19:38:36,137] {standard_task_runner.py:80} INFO - Job 764: Subtask second_task
[2022-08-06 19:38:36,228] {task_command.py:371} INFO - Running <TaskInstance: our_first_dag_v5.second_task scheduled__2022-04-08T00:00:00+00:00 [running]> on host b29c168c1666
[2022-08-06 19:38:36,327] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2022-04-08T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-04-08T00:00:00+00:00
[2022-08-06 19:38:36,327] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-08-06 19:38:36,328] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2022-08-06 19:38:36,345] {subprocess.py:85} INFO - Output:
[2022-08-06 19:38:36,347] {subprocess.py:92} INFO - hey, I am task2 and will be running after task1!
[2022-08-06 19:38:36,347] {subprocess.py:96} INFO - Command exited with return code 0
[2022-08-06 19:38:36,431] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20220408T000000, start_date=20220806T193836, end_date=20220806T193836
[2022-08-06 19:38:36,475] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-06 19:38:36,534] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-08-06 21:52:27,584] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-04-08T00:00:00+00:00 [queued]>
[2022-08-06 21:52:27,595] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-04-08T00:00:00+00:00 [queued]>
[2022-08-06 21:52:27,595] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 21:52:27,595] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2022-08-06 21:52:27,595] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 21:52:27,615] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): second_task> on 2022-04-08 00:00:00+00:00
[2022-08-06 21:52:27,621] {standard_task_runner.py:52} INFO - Started process 2674 to run task
[2022-08-06 21:52:27,624] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v5', 'second_task', 'scheduled__2022-04-08T00:00:00+00:00', '--job-id', '762', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmpb1363nj6', '--error-file', '/tmp/tmpinj7zpml']
[2022-08-06 21:52:27,624] {standard_task_runner.py:80} INFO - Job 762: Subtask second_task
[2022-08-06 21:52:27,713] {task_command.py:371} INFO - Running <TaskInstance: our_first_dag_v5.second_task scheduled__2022-04-08T00:00:00+00:00 [running]> on host c3f731879890
[2022-08-06 21:52:27,853] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2022-04-08T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-04-08T00:00:00+00:00
[2022-08-06 21:52:27,854] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-08-06 21:52:27,855] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2022-08-06 21:52:27,874] {subprocess.py:85} INFO - Output:
[2022-08-06 21:52:27,876] {subprocess.py:92} INFO - hey, I am task2 and will be running after task1!
[2022-08-06 21:52:27,877] {subprocess.py:96} INFO - Command exited with return code 0
[2022-08-06 21:52:27,920] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20220408T000000, start_date=20220806T215227, end_date=20220806T215227
[2022-08-06 21:52:27,956] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-06 21:52:27,995] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-08-06 21:57:53,370] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-04-08T00:00:00+00:00 [queued]>
[2022-08-06 21:57:53,384] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-04-08T00:00:00+00:00 [queued]>
[2022-08-06 21:57:53,384] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 21:57:53,384] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2022-08-06 21:57:53,384] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 21:57:53,399] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): second_task> on 2022-04-08 00:00:00+00:00
[2022-08-06 21:57:53,405] {standard_task_runner.py:52} INFO - Started process 2468 to run task
[2022-08-06 21:57:53,407] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v5', 'second_task', 'scheduled__2022-04-08T00:00:00+00:00', '--job-id', '762', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmp7m0u97b7', '--error-file', '/tmp/tmp3xy7f3xi']
[2022-08-06 21:57:53,407] {standard_task_runner.py:80} INFO - Job 762: Subtask second_task
[2022-08-06 21:57:53,474] {task_command.py:371} INFO - Running <TaskInstance: our_first_dag_v5.second_task scheduled__2022-04-08T00:00:00+00:00 [running]> on host bc7e154a4fdd
[2022-08-06 21:57:53,556] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2022-04-08T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-04-08T00:00:00+00:00
[2022-08-06 21:57:53,557] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-08-06 21:57:53,558] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2022-08-06 21:57:53,571] {subprocess.py:85} INFO - Output:
[2022-08-06 21:57:53,572] {subprocess.py:92} INFO - hey, I am task2 and will be running after task1!
[2022-08-06 21:57:53,572] {subprocess.py:96} INFO - Command exited with return code 0
[2022-08-06 21:57:53,604] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20220408T000000, start_date=20220806T215753, end_date=20220806T215753
[2022-08-06 21:57:53,659] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-06 21:57:53,704] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-08-06 22:12:55,719] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-04-08T00:00:00+00:00 [queued]>
[2022-08-06 22:12:55,729] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-04-08T00:00:00+00:00 [queued]>
[2022-08-06 22:12:55,729] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 22:12:55,729] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2022-08-06 22:12:55,729] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 22:12:55,741] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): second_task> on 2022-04-08 00:00:00+00:00
[2022-08-06 22:12:55,752] {standard_task_runner.py:52} INFO - Started process 2484 to run task
[2022-08-06 22:12:55,757] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v5', 'second_task', 'scheduled__2022-04-08T00:00:00+00:00', '--job-id', '764', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmpmshi56a9', '--error-file', '/tmp/tmpom08ybb4']
[2022-08-06 22:12:55,757] {standard_task_runner.py:80} INFO - Job 764: Subtask second_task
[2022-08-06 22:12:55,845] {task_command.py:371} INFO - Running <TaskInstance: our_first_dag_v5.second_task scheduled__2022-04-08T00:00:00+00:00 [running]> on host 1250c3b659cd
[2022-08-06 22:12:55,953] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2022-04-08T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-04-08T00:00:00+00:00
[2022-08-06 22:12:55,955] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-08-06 22:12:55,956] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2022-08-06 22:12:55,974] {subprocess.py:85} INFO - Output:
[2022-08-06 22:12:55,975] {subprocess.py:92} INFO - hey, I am task2 and will be running after task1!
[2022-08-06 22:12:55,976] {subprocess.py:96} INFO - Command exited with return code 0
[2022-08-06 22:12:56,015] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20220408T000000, start_date=20220806T221255, end_date=20220806T221256
[2022-08-06 22:12:56,048] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-06 22:12:56,090] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-08-06 22:24:13,478] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-04-08T00:00:00+00:00 [queued]>
[2022-08-06 22:24:13,493] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-04-08T00:00:00+00:00 [queued]>
[2022-08-06 22:24:13,493] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 22:24:13,493] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2022-08-06 22:24:13,493] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 22:24:13,510] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): second_task> on 2022-04-08 00:00:00+00:00
[2022-08-06 22:24:13,517] {standard_task_runner.py:52} INFO - Started process 2536 to run task
[2022-08-06 22:24:13,520] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v5', 'second_task', 'scheduled__2022-04-08T00:00:00+00:00', '--job-id', '766', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmpcvbbp1xm', '--error-file', '/tmp/tmpy9qvpkze']
[2022-08-06 22:24:13,520] {standard_task_runner.py:80} INFO - Job 766: Subtask second_task
[2022-08-06 22:24:13,592] {task_command.py:371} INFO - Running <TaskInstance: our_first_dag_v5.second_task scheduled__2022-04-08T00:00:00+00:00 [running]> on host b215076695c3
[2022-08-06 22:24:13,679] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2022-04-08T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-04-08T00:00:00+00:00
[2022-08-06 22:24:13,680] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-08-06 22:24:13,681] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2022-08-06 22:24:13,696] {subprocess.py:85} INFO - Output:
[2022-08-06 22:24:13,698] {subprocess.py:92} INFO - hey, I am task2 and will be running after task1!
[2022-08-06 22:24:13,698] {subprocess.py:96} INFO - Command exited with return code 0
[2022-08-06 22:24:13,732] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20220408T000000, start_date=20220806T222413, end_date=20220806T222413
[2022-08-06 22:24:13,771] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-06 22:24:13,816] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
